### ISSUE TITLE
Train Basic CNN AutoEncoder to generate Sprites

### ISSUE THREAD

**Issue by flyxiv** [flyxiv/nino#8]
For starter, implementing a CNN AutoEncoder to baseline the generator performance.

**Comment by flyxiv** [comment]
![Image](https://github.com/user-attachments/assets/421045ef-5cd7-442f-8543-552ae618b9e1)

Created a CNN-Base AutoEncoder

**Comment by flyxiv** [comment]
![Image](https://github.com/user-attachments/assets/8998d3df-19da-4dbb-b089-6c873ff05500)
Does try to generate the trace of the wanted shape, but loses resolution.

**Comment by flyxiv** [comment]
Colors are also fuzzy, this is probably why we use max pooling in CNN chained layers.

**Comment by flyxiv** [comment]
Trained for 400 epochs(previously 100). Now it generates training set pretty accurately.

* Hossabara from Triangle Tactics

label:

![Image](https://github.com/user-attachments/assets/9b983dd0-1e9b-4274-8001-1f350488c229)

generated:

![Image](https://github.com/user-attachments/assets/1a354d3c-9fae-4ddc-a527-51329b0dd53a)

**Comment by flyxiv** [comment]
![Image](https://github.com/user-attachments/assets/b684462e-aef0-4cbe-9a9d-2b374a2ae450)

Tried a test set that wasn't trained on. The model is not generalized enough to create a reasonable generation for untrained images yet.

Good enough for a baseline. I should implement cross validation training since the sample size is small.
